packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testOpenPreferences.st

```diff
@@ -0,0 +1,16 @@
+as yet unclassified
+testOpenPreferences
+
+	| prefBrowser prefBrowserMorph |
+
+	klass new openPreferences.
+
+	prefBrowserMorph := [World submorphs detect: [ :morph | morph class name = #PreferenceBrowserMorph]]
+		on: NotFound
+		do: [:exc | Transcript show: exc messageText; cr. ^nil].
```

Comment:
yes but if you return nil in a test it passes and doesn't fail. you need an assertion which fails